generator/views/lookml_utils.py

```diff
@@ -117,6 +117,16 @@ def _generate_dimensions(client: bigquery.Client, table: str) -> List[Dict[str,
     """
     dimensions = {}
     for dimension in _generate_dimensions_helper(client.get_table(table).schema):
+        # Rename `event` field in `events_stream` to `full_event_name` to expose it in Looker views.
+        # This is required since `event_timestamp` is renamed to `event` in `_get_dimension` and takes precedence below.
+        if (
+            table.endswith(".events_stream")
+            and dimension["name"] == "event"
+            and dimension["type"] == "string"
+        ):
+            # `event` field contains concatenated event category and name
+            dimension["name"] = "full_event_name"
```

Comment:
Can the `and not (name == "event" and dimension["type"] == "time")` line below be removed since this is being renamed?  Looks like that's just for events stream https://github.com/mozilla/lookml-generator/pull/854 and might be useful to catch other similar issues.  Not critical though and maybe it's better to leave it as-is